Formula/qt.rb

```diff
@@ -19,6 +19,7 @@ class Qt < Formula
 
   keg_only "Qt 5 has CMake issues when linked"
 
+  depends_on :macos => :sierra
```

Comment:
audit was complaining to move this _below_ the other `depends_on`s.

Reply:
fixed in 61fb406f582d7692cbeff46b538bcec1d1152ee4